pic-sure-auth-services/src/main/java/edu/harvard/hms/dbmi/avillach/auth/service/auth/FENCEAuthenticationService.java

```diff
@@ -86,6 +88,13 @@ public class FENCEAuthenticationService {
     		"\\\\_Consents\\\\"   ///old _Consents\Short Study... path no longer used, but still present in examples.
     };
 
+    //TODO: Remove when RAS is implemented
+    private static final List<String> openDatasets = Collections.unmodifiableList(
+        new ArrayList<>() {{
+            add("tutorial-biolincc_camp");
+            add("tutorial-biolincc_digitalis");
+    }});
```

Comment:
`private static final List<String> openDatasets = List.of("tutorial-biolincc_camp", "tutorial-biolincc_digitalis");

Reply:
Oh crap this is from yesterday. Whatever. Point stands.